net/getting-started/security/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Licensing
+type: docs
+weight: 160
+url: /net/security/
+---
+
+## **Security in Aspose.Slides**
+
+Aspose applies the best industrial practices when developing its products. 
+
+* Aspose.Slides for .NET is used to manipulate and transform presentations in files in other formats. It does not run scripts in presentations. Aspose.Slides simply parses the presentation structure and allows the end-user's code to manipulate the object model in a convenient way. 
+* Aspose.Slides functions as a library in the context of parsing and interpreting documents without the execution of remote code. All Aspose products are executed on your machines and network. They do not transmit any data to Aspose. The only exception exists when a [metered license](https://purchase.aspose.com/faqs/licensing/metered) is involved: if you use a metered licensed, your API usage information is processed.
```

Comment:
>products are executed on your machines and network

Here I have the impression that Slides has something to do with your machines and network. I would leave out the network part.

And in fact, Slides by default loads pictures via links from the web (or local network, or path on the machine) - just like PowerPoint does. It is possible to override this behavior through the implementation of the IResourceLoadingCallback interface (https://reference.aspose.com/words/net/aspose.words.loading/iresourceloadingcallback/).

Reply:
I have rewritten that part to remove "network". Thank you.